pubspec.yaml

```diff
@@ -108,6 +108,13 @@ dependencies:
   app_links: ^3.4.3
   share_plus: ^6.2.0
 
+  # Using the forked copy of receive_sharing_intent because of the gradle version issue
```

Comment:
can you try to figure out whichever gradle issue this is. We can't keep on using forked repo

Reply:
It's because of the official receive_sharing_intent package is not updated to the latest version of Kotlin.

ISSUE - https://github.com/KasemJaffer/receive_sharing_intent/issues/256#issuecomment-1565167860
PR - https://github.com/KasemJaffer/receive_sharing_intent/pull/255